Given some RNA-seq data in FASTA format, find the vector of equivalence class counts.
Your implementation should in the least do the following:
- Given a gene annotation, an index should be produced which can be read in by your pseudoalignment procedure.
- Your pseudoalignment procedure should take in (1) the previously mentioned index, (2) RNA-seq data and output equivalence class counts, and (3) a k-mer length.

Provide some basic statistics about the size of the equivalence classes and how much data is mapping to them. This sort of summary can be provided in a figure/plot.

* Note: reads.fasta was too large to upload to this repository
